automatic_symbol_loading/automatic_symbol_loading.py

```diff
@@ -0,0 +1,72 @@
+from pathlib import Path
+
+import gdb
+
+try:
+    from elftools.elf.elffile import ELFFile
+except ModuleNotFoundError:
+    check_build_id = False
+else:
+    check_build_id = True
+
+
+def create_file_dict(path: str) -> dict:
```

Comment:
Did you revert this to be a string? I thought you modified it to be a `Path`

Reply:
You should indicate also the type in the dictionary (`Dict[str, Path]`).

Reply:
This is the function that changes the str and makes it become a Path, as it accepts in input the string passed by the user.
Adding the type int for the dictionary.